browser/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../base/.eslintrc.json",
+  "globals": {
+    "window": false,
```

Comment:
There have been numerous bugs caused by the browser env on multiple projects. We should manually define these otherwise we have `event` and `name` and god only knows what else that'll cause bugs.